docs/tutorial_bracket_strategy.md

```diff
@@ -140,37 +140,45 @@ yarn run networks-inject
 
-Then it will create a transaction deploying new safes. All the address of the gnosis-safes will be printed on the screen.
+Then, it will create a transaction deploying new safes. All the address of the Gnosis-Safes will be printed on the screen.
 
 Afterwards, the transaction containing all the orders will be build.
-All the order placements will be bundled into one transaction, which will appear first in the gnosis-interface.
-Before you sign the transaction, make sure to check the script output for the displayed orders and check it for correctness.
+All the order placements will be bundled into one transaction, which will appear first in the Gnosis-Interface.
+Before you sign the transaction, make sure to check the correctness of the displayed orders of the script output.
 
 Also, the transaction sending the DAI and WETH into the brackets and depositing it on behalf of the brackets into the GP exchange contract is built in parallel.
 This is the second transaction send to the interface.
 
-If everything went smooth, you should see the two transaction initiations confirmed like that:
+If everything goes smoothly, you should see the two transaction initiations confirmed like that:
 
 ```ssh
+
+==> Sending the order placing transaction to gnosis-safe interface.
+    Attention: This transaction MUST be executed first!
+
+Signing and posting multi-send transaction 0x007ecc58f9dc222c26215b260abc363e0308dfec539d7eeabb8a2c9f09397157 from proposer account 0x740a98F8f4fAe0986FB3264Fe4aaCf94ac1EE96f
+Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
+
 ==> Sending the funds transferring transaction, please execute this transaction second
+
 Signing and posting multi-send transaction 0x09cb78a5a49f10305a2f108d45e8fa059e4231ebf219315c56409d67419d65fb from proposer account 0x740a98F8f4fAe0986FB3264Fe4aaCf94ac1EE96f
 Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
 ```
 
-Now, you should just follow the link and sign & execute the lower transaction first, the one which was initiated at first. It should look like this:
+Now, you should just follow the link and sign & execute the transaction with the lower transaction-ID first. It should look like this:
 
 ![Read tokenAddressToIdMap](assets/tutorial_bracket_strategy_signing.png)
 
-For executing, just press the button execute.
+For executing, just press the button _confirm_.
 
 If you are executing the second transaction with a lower gasPrice than the first one, then you can sign it immediately.
 Otherwise, you should wait until the first one is mined. This works as the gas price usually determines the mining order of the transaction.
 
 Congrats. You managed to provide liquidity and have a chance to earn a passive income, if the prices are moving and returning back to your initial currentPrice.
```

Comment:
I am not entirely sure how OK this is... Also depends on that orders are actually matched...?